superset/db_engine_specs.py

```diff
@@ -978,6 +978,14 @@ def get_column_names(filepath):
                 return next(unicodecsv.reader(f, encoding='utf-8-sig'))
 
         table_name = form.name.data
+        if config.get('UPLOADED_CSV_HIVE_NAMESPACE'):
+            if  '.' in table_name:
+                raise Exception(
+                    """You can't specify a namespace. """
```

Comment:
If you’re going to use “”” which reprsents a multiline string then make the string multiline. It’s probably more desirable to use single quoted strings and wrap either via \ or encapsulate in ().